module1/lessons/inheritance.md

```diff
@@ -103,29 +119,38 @@ ali = Ceo.new(15, 20)
 puts ali.total_compensation
 ```
 
+<section class="dropdown">
+### Putting It All Together
+
 You can see that even though we didn't define `total_compensation` in our `Ceo` class, the `Ceo` object still responded to this methods because it inherited them from `Employee`. In this example, `Ceo` is the **subclass** or **child**, and `Employee` is the **superclass** or **parent**.
 
 We can still define methods in our `Ceo` class just like with any other class.
+</section>
 
-**NOTE**: It's tempting to think of inheritance as methods getting "passed down" to a child (as you would when thinking about genetics in living creatures); however, technically, the method calls on the child object are "passed up" to the parent class. When a method is called on the child that only exists in the parent class, that message is automatically delegated to the parent class.
+<section class="note">
+### NOTE:
 
+It's tempting to think of inheritance as methods getting "passed down" to a child (as you would when thinking about genetics in living creatures); however, technically, the method calls on the child object are "passed up" to the parent class. When a method is called on the child that only exists in the parent class, that message is automatically delegated to the parent class.
```

Comment:
Is the word "message" in the last sentence supposed to be "method"?

Reply:
I think "message" works here, since the act of calling methods is basically passing messages.... at least that's my understanding.

Reply:
Thanks Kat!